Cargo.toml

```diff
@@ -90,9 +90,13 @@ static_assertions = "1.1"
 thiserror = "1.0"
 tokio = "1.0"
 
+[profile.dev]
+panic = "abort"
```

Comment:
Please add a comment here explaining why this is being added. I'm sure in a couple of months we'll forget.